cluster/gce/config-default.sh

```diff
@@ -140,9 +140,8 @@ if [[ ${NETWORK_POLICY_PROVIDER:-} == "calico" ]]; then
 	NODE_LABELS="${NODE_LABELS},projectcalico.org/ds-ready=true"
 fi
 
-# Turn the simple metadata proxy on by default.
-ENABLE_METADATA_PROXY="${ENABLE_METADATA_PROXY:-simple}"
```

Comment:
Since the possible values here aren't simply true/false, it might be worth leaving in a line that sets this var blank with a comment that explains the valid values.

Reply:
Done.